ms2deepscore/train_new_model/spectrum_pair_selection.py

```diff
@@ -150,13 +152,14 @@ def compute_fingerprint_dataframe(
         The settings that should be used for selecting the compound pairs wrapper. The settings should be specified as a
         SettingsMS2Deepscore object.
     """
-    if settings.random_seed is not None:
-        np.random.seed(settings.random_seed)
+    # todo why do we need to set a random seed here??
```

Comment:
@florian-huber Why would we need a random seed here? The fingerprint calculations don't have a random component right?

Reply:
True. Must be a leftover and can be removed!